models/constants.py

```diff
@@ -0,0 +1,125 @@
+SEEDS_TO_COUNT = [
+    'kmer8_count',
+    'kmer7_count',
+    'kmer6_count',
+    'kmer6_bulge_count',
+    'kmer6_mismatch_count',
+]
+
+SEED_COUNT_COLUMNS = [
+    'kmer8_count_1', 'kmer8_count_2',
+       'kmer8_count_3', 'kmer8_count_4', 'kmer8_count_5', 'kmer8_count_6',
+       'kmer7_count_1', 'kmer7_count_2', 'kmer7_count_3', 'kmer7_count_4',
+       'kmer7_count_5', 'kmer7_count_6', 'kmer6_count_1', 'kmer6_count_2',
+       'kmer6_count_3', 'kmer6_count_4', 'kmer6_count_5', 'kmer6_count_6',
+       'kmer6_bulge_count_1', 'kmer6_bulge_count_2', 'kmer6_bulge_count_3',
+       'kmer6_bulge_count_4', 'kmer6_bulge_count_5', 'kmer6_bulge_count_6',
+       'kmer6_bulge_or_mismatch_count_1', 'kmer6_bulge_or_mismatch_count_2',
+       'kmer6_bulge_or_mismatch_count_3', 'kmer6_bulge_or_mismatch_count_4',
+       'kmer6_bulge_or_mismatch_count_5', 'kmer6_bulge_or_mismatch_count_6',
+]
+
+TARGETSCAN_COLUMN_TO_SEQUENCE = {
```

Comment:
Providing links for all sequences, where you took them form, would be great.

Reply:
Done